fortinet-fortirecon-aci/info.json

```diff
@@ -1795,178 +1364,84 @@
-      "description": "Retrieves a list Vulnerability Intelligence CVEs for the given organization ID and the input parameters specified.",
+      "description": "Retrieves vendor watch-list for the given organization ID.",
       "category": "investigation",
       "is_config_required": true,
       "visible": true,
       "enabled": true,
       "parameters": [
         {
-          "name": "fortirecon_severity",
-          "title": "Fortirecon Severity",
-          "type": "select",
+          "name": "domain",
+          "title": "Filter By Domain",
+          "type": "text",
           "editable": true,
           "visible": true,
           "required": false,
-          "tooltip": "Select the severity based on which you want to filter the result. You can select between CRITICAL, HIGH, MEDIUM and LOW.",
-          "description": "Select the severity based on which you want to filter the result. You can select between CRITICAL, HIGH, MEDIUM and LOW.",
-          "options": [
-            "CRITICAL",
-            "HIGH",
-            "MEDIUM",
-            "LOW"
-          ]
+          "tooltip": "Specify a domain to filter the vendors added for monitoring by domain.",
+          "description": "Specify a domain to filter the vendors added for monitoring by domain."
         },
         {
-          "name": "cve_year",
-          "title": "CVE Year",
+          "name": "name",
+          "title": "Filter By Name",
           "type": "text",
           "editable": true,
           "visible": true,
           "required": false,
-          "tooltip": "Filter the results by specific CVE year. Multiple CVE years should be comma separated and will work as OR filter, example: 2018,2019.",
-          "description": "Filter the results by specific CVE year. Multiple CVE years should be comma separated and will work as OR filter, example: 2018,2019."
+          "tooltip": "Specify a name to filter the vendors added for monitoring by name.",
+          "description": "Specify a name to filter the vendors added for monitoring by name."
         },
         {
-          "name": "addition",
-          "title": "Addition",
+          "name": "approval_status",
+          "title": "Approval Status",
           "type": "select",
           "editable": true,
           "visible": true,
           "required": false,
-          "tooltip": "Select the addition type based on which you want to filter the result. You can select between EASM and MANUAL.",
-          "description": "Select the addition type based on which you want to filter the result. You can select between EASM and MANUAL.",
+          "tooltip": "Specify the approval status of the vendor, you can select between pending, approved, rejected.",
+          "description": "Specify the status of the vendor, you can select between pending, approved, rejected.",
           "options": [
-            "EASM",
-            "MANUAL"
+            "pending",
```

Comment:
Capitalize the first letters of every option (e.g. Pending) do this which every parameters

Reply:
Done